src/data/print_documents/documents.py

```diff
@@ -1,7 +1,7 @@
 from abc import ABC, abstractmethod
 from functools import cached_property, partial
 import itertools
-from typing import Any, override
+from typing import Any, Tuple, override
```

Comment:
Better use built-in ``tuple`` no?
```suggestion
from typing import Any, override
```